extension/test/integration/config-loader.spec.js

```diff
@@ -0,0 +1,50 @@
+const fs = require('fs')
+const homedir = require('os').homedir()
+
+describe('::config-loader::', () => {
+  it(
+    'when ADYEN_INTEGRATION_CONFIG is not set but external file is configured, ' +
+      'then it should load configuration correctly',
+    () => {
+      const filePath = `${homedir}/.extensionrc`
+      try {
+        delete process.env.ADYEN_INTEGRATION_CONFIG
+        const config = {
+          commercetools: {
+            ctpProjectKey1: {
+              clientId: 'clientId',
+              clientSecret: 'clientSecret',
+              apiUrl: 'host',
+              authUrl: 'authUrl',
+              authentication: {
+                scheme: 'basic',
+                username: 'username',
+                password: 'password',
+              },
+            },
+          },
+          adyen: {
+            adyenMerchantAccount1: {
+              apiBaseUrl: 'apiBaseUrl',
+              apiKey: 'apiKey',
+              clientKey: 'clientKey',
+              legacyApiBaseUrl: 'legacyApiBaseUrl',
+            },
+          },
+          logLevel: 'DEBUG',
+        }
+        fs.writeFileSync(filePath, JSON.stringify(config), 'utf-8')
+
+        requireUncached('../../src/config/config')
```

Comment:
Nothing is expected here. Maybe after loading the config, you can check that the config has one of the values you expected.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/870/commits/ca80ed8f0fd3fe76cfb4b94c4ffab76bae454e94